data/json/items/tools.json

```diff
@@ -8316,5 +8316,19 @@
     "symbol": ",",
     "color": "light_gray",
     "flags": [ "NO_SALVAGE" ]
+  },
+  {
+    "id": "electrolysis_kit",
+    "type": "TOOL",
+    "name": "electrolysis kit",
+    "description": "A set of wiring and electrodes for applying a direct current, usually to a liquid. Useful for crafting. Load with a storage battery or 12V vehicle battery to use.",
```

Comment:
Please use two spaces after periods.

Reply:
Ok thanks, will try to remember that in future.